rel/i18n/emqx_conf_schema.hocon

```diff
@@ -475,6 +475,19 @@ log_burst_limit_window_time.desc:
 log_burst_limit_window_time.label:
 """Window Time"""
 
+desc_log_throttling.label:
+"""Log Throttling"""
+
+desc_log_throttling.desc:
+"""Log throttling feature reduces the number of potentially flooding logged events by
+dropping all but the first event within a configured time window."""
+
+log_throttling_window_time.desc:
+"""A time interval at which log throttling is applied. Defaults to 1 minute."""
```

Comment:
no need to mention default value in desc, the schema dump generates default along side it.
```suggestion
"""For throttled messages, only log 1 in each time window."""
```